src/main/typescript/components/RatingTable.tsx

```diff
@@ -2,12 +2,47 @@ import * as AppReducer from "./AppReducer"
 import { formatDate } from "../Util"
 
 import * as React from "react"
+import styled from 'styled-components'
 
 interface Props {
   dispatch: AppReducer.Dispatch,
   state: AppReducer.State
 }
 
+function getWidthPercent(score: number, maxScore: number): number {
+
```

Comment:
There's a probably unintentional blank line here, and some rogue tab characters below.

Reply:
@vtan bad news: it wasn't unintentional at all, leaving a newline before return statements is just a convention I learned god knows where. I removed it tho.